csob/api.py

```diff
@@ -287,4 +321,38 @@ def customer_info(self, customer_id: str) -> APIResponse:
         Returns:
             APIResponse
         """
-        raise NotImplementedError()
+        return CustomerInfoResource(**self.get_resource_kwargs()).get(self._get_private_key(), customer_id)
+
+    def _get_private_key(self) -> str:
```

Comment:
Používá se to celkem často, nebylo by dobré to cachovat než pokaždé číst soubor z disku znovu?

Reply:
@rubickcz říkal jsem si že privatní klíč chceme mít v ramce co nejměně je potřeba